rewriter/Visibility.cc

```diff
@@ -0,0 +1,46 @@
+#include "rewriter/Visibility.h"
+#include "ast/Helpers.h"
+#include "ast/ast.h"
+#include "core/Context.h"
+#include "core/Names.h"
+#include "core/core.h"
+#include "core/errors/rewriter.h"
+#include "rewriter/rewriter.h"
+
+using namespace std;
+
+namespace sorbet::rewriter {
+
+void Visibility::run(core::MutableContext ctx, ast::ClassDef *classDef) {
+    core::NameRef currentVisibility = core::Names::public_();
+
+    for (auto &stat : classDef->rhs) {
+        typecase(
+            stat.get(), [&](ast::Assign *assign) {},
```

Comment:
```suggestion
            stat.get(),
```

This is redundant with the fallback case below for `ast::Expression`, right?

Reply:
Is `ast::Assign` a subtype of `ast::Expression`? That is, will it also be considered to satisfy that case in the typecase?

Reply:
Yep.

Reply:
Pushed this change.